src/com/spikes2212/dashboard/ConstantHandler.java

```diff
@@ -63,8 +79,24 @@
      * If it cannot communicate a value from the {@link SmartDashboard}, it will return an empty string.
      */
     public static Supplier<String> addConstantString(String name, String value) {
-        SmartDashboard.putString(name, value);
-        return () -> SmartDashboard.getString(name, value);
+    	if (!Preferences.getInstance().containsKey(name)) {
+    		Preferences.getInstance().getString(name, value);
+    		stringMap.put(name, value);
+            return () -> Preferences.getInstance().getString(name, value);
+    	}
+    	return () -> Preferences.getInstance().getString(name, value);
+    }
+    
+    public static void reset() {
+    	for (String key : doubleMap.keySet()) {
+    		Preferences.getInstance().putDouble(key, doubleMap.get(key));

```

Comment:
make sure writing a new value to the same key overrides the old value, though it probably does